project/libs/publication/like-module/src/lib/like.service.ts

```diff
@@ -1,35 +1,58 @@
-import { Injectable, NotFoundException } from '@nestjs/common';
+import { Injectable, NotFoundException, ConflictException } from '@nestjs/common';
 import { BaseMemoryRepository } from '@project/data-access';
 import { LikeEntity } from './like.entity';
+import { Like } from './like.interface';
 import { LikeDto } from './dto/like.dto';
+import { PrismaClientService } from '@project/models';
 
 @Injectable()
-export class LikeService extends BaseMemoryRepository<LikeEntity> {
+export class LikeService {
+  constructor (
+    	private readonly client: PrismaClientService
+  ) {}
```

Comment:
Service не должен взаимодействовать с Prisma в прямую, только через Repository.

```suggestion
  constructor (
    	private readonly likeRepository: LikeRepository
  ) {}
```